rackcity/permissions/permissions.py

```diff
@@ -75,48 +75,90 @@ def get_permission(permission_name: PermissionName) -> Permission:
     return permission
 
 
-def user_has_asset_permission(user, site=None):
-    if user.has_perm(PermissionPath.ASSET_WRITE.value):
-        return True
-    if site:
+def user_has_asset_permission(user, site):
+    return user.has_perm(
```

Comment:
Generally opposed to abbreviations in code but I like `user.has_perm` bc it makes me think of a user having a perm like the hair style so I'll let it slide

Reply:
HAHAHA please take this up with Django, sir. I did not write the method.